x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_enrichment/processors/grok/grok_ai_suggestions.tsx

```diff
@@ -318,12 +330,42 @@ function InnerGrokAiSuggestions({
 }
 
 export function GrokAiSuggestions() {
-  const isAiEnabled = useAiEnabled();
+  const {
+    core: { http },
+  } = useKibana();
+  const { enabled: isAiEnabled, couldBeEnabled } = useAiEnabled();
   const props = useSimulatorContext();
 
+  if (!isAiEnabled && couldBeEnabled) {
+    return (
+      <EuiToolTip
+        content={i18n.translate(
+          'xpack.streams.streamDetailView.managementTab.enrichment.processorFlyout.aiAssistantNotEnabledTooltip',
+          {
+            defaultMessage:
+              'AI Assistant features are not enabled. To enable features, add an AI connector on the management page.',
+          }
+        )}
+      >
+        <EuiButton
+          iconType="sparkles"
+          href={http!.basePath.prepend(
+            `/app/management/insightsAndAlerting/triggersActionsConnectors/connectors`
+          )}
+        >
```

Comment:
Couple of notes on this:
- Should we show it as a link since it navigates on a different page? The button feels more like "Click and enable".
- Should we also open it in a new tab? If there is some work in progress the user will get prompted to discard changes and leave the page, which might not be desirable.